x/proof/types/claim.go

```diff
@@ -42,6 +42,25 @@ func (claim *Claim) GetNumEstimatedComputeUnits(
 	return new(big.Int).Div(numerator, denominator).Uint64(), nil
 }
 
+// GetNumEstimatedRelays returns the claim's estimated number of relays.
+func (claim *Claim) GetNumEstimatedRelays(
```

Comment:
Could/should we share this business logic in `scaleRelaysFromActualToTarget`?

I'm surprised we haven't had a similar function already.

Reply:
I added a `protocol` package function called `GetNumEstimatedRelays`, but I don’t think it can be consolidated with `scaleRelaysFromActualToTarget`. The reason is that `GetNumEstimatedRelays` calculates the *estimated relays* based on *mined relays* and *mining difficulty*, while `scaleRelaysFromActualToTarget` computes the *volume of applicable relays* using *relay probability* and *estimated relays*.